.changeset/thirty-owls-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': patch
+---
+
+# exposes undocumented `PageTheme`
```

Comment:
you can remove the `#` header here. Otherwise it could look strange in the release log :)

Reply:
done